pygaggle/rerank/transformer.py

```diff
@@ -30,7 +30,7 @@ def rerank(self, query: Query, texts: List[Text]) -> List[Text]:
             attn_mask = batch.output['attention_mask']
             _, batch_scores = greedy_decode(self.model,
                                             input_ids.to(self.device),
-                                            length=2,
+                                            length=1,
```

Comment:
I can't remember exactly why, but @daemon had a good reason to use length=2, so I will let him comment here

Reply:
The reason was to reproduce what the TensorFlow implementation had been doing. If using a single step of decoding improves quality, then that's all the better.

Reply:
Adding a comment would be a good idea...

Reply:
Sure, that belongs in covidex.